engine/src/flutter/impeller/display_list/canvas.cc

```diff
@@ -540,6 +540,16 @@ void Canvas::DrawRoundRect(const RoundRect& round_rect, const Paint& paint) {
   DrawPath(path, paint);
 }
 
+void Canvas::DrawRoundSuperellipse(const RoundSuperellipse& rse,
+                                   const Paint& paint) {
+  Entity entity;
+  entity.SetTransform(GetCurrentTransform());
+  entity.SetBlendMode(paint.blend_mode);
+
+  RoundSuperellipseGeometry geom(rse.GetBounds(), rse.GetRadii());
+  AddRenderEntityWithFiltersToCurrentPass(entity, &geom, paint);
```

Comment:
Also, this method looks suspiciously simpler than the similar DrawRRect which only uses the Entity/Geometry for filled paths with all the same corners and otherwise redirects to drawing a path. I haven't looked at the entity/geometry code yet to see how all that is supposed to work, though.

Reply:
Yeah I was about to change it. I've added stroking to this method. This PR is supposed to allow impeller to both draw and stroke RSEs.